src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts

```diff
@@ -222,11 +221,10 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
 					}
 				}]
 			},
-			'extensions.experimental.useUtilityProcess': {
+			'extensions.experimental.useUtilityProcess': { // TODO@bpasero remove me once sandbox is enabled by default
 				type: 'boolean',
-				tags: ['experimental'],
 				description: localize('extensionsUseUtilityProcess', "When enabled, the extension host will be launched using the new UtilityProcess Electron API."),
-				default: product.quality === 'stable' ? false : true // disabled by default in stable for now
+				default: true
```

Comment:
Will this still be disabled for OSS ?

Reply:
Oh yeah, I forgot about https://github.com/microsoft/vscode/blob/ed8b63d8bf677cef75f3c5f1d183eac2d09ab3d8/src/vs/base/parts/sandbox/electron-main/electronTypes.ts#L137